### PR TITLE
Make it more consistent with the API

### PR DESCRIPTION
**Pull request by sakisv**
Copy things from notifications-api to ensure that we're starting
properly and we're shipping logs to cloudwatch

----

Need to double check that we're exporting logs in the correct format and make sure we test it manually on preview before it gets merged.

### CODE REVIEW

**Comment by leohemsted** [comment]
we're definitely not exporting logs in the right format

**Comment by leohemsted** [comment]
app/__init__.py should probably say something like

```
from notifications_utils import logging

def create_app():
   [...]
    logging.init_app(application, statsd_client)
```

at line 68-70

File: requirements.txt [hunk 1]
```diff
@@ -16,5 +16,5 @@ git+https://github.com/alphagov/notifications-utils.git@23.7.0#egg=notifications
 
 # PaaS requirements
 gunicorn==19.7.1
-# awscli>=1.11,<1.12
-# awscli-cwlogs>=1.4,<1.5
+awscli>=1.11,<1.12
```
Comment (leohemsted) [inline requirements.txt]: we don't pin `awscli` in the other repos 🤔

File: app/gunicorn_config.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import os
+import sys
+import traceback
+
+workers = 10
+errorlog = "/home/vcap/logs/gunicorn_error.log"
```
Comment (leohemsted) [inline app/gunicorn_config.py]: do we want to move the dockerfile working directory so that the app is in `/home/vcap/app`?

Comment (sakisv) [reply app/gunicorn_config.py]: I don't see why not 👍

**Pull request merged**